dashboards/process_memory/main.yml

```
@@ -0,0 +1,33 @@
+metric_keys:
+  - process_rss
+dashboard:
+  title: Process memory usage
```

```suggestion
  title: Process memory usage
  description: RSS memory usage of your processes split out per host and process type.
```

seemed to have gone missing after a force push?